prediction_market_agent/data_models/market_data_models.py

```diff
@@ -1,16 +1,38 @@
+import typing as t
 from pydantic import BaseModel
 from web3 import Web3
 from web3.types import Wei
 from prediction_market_agent.tools.types import (
     USD,
-    OmenOutcomeToken,
-    xDai,
     HexAddress,
     ChecksumAddress,
+    Probability,
+    Mana,
+    OmenOutcomeToken,
+    xDai,
 )
+from datetime import datetime
+
 
+class AgentMarket(BaseModel):
+    """
+    Common market class that can be created from vendor specific markets.
+    Contains everything that is needed for an agent to make a prediction.
+    """
+
+    id: str
+    question: str
+    outcomes: list[str]
+    original_market: t.Union["OmenMarket", "ManifoldMarket"]
```

Comment:
Don't think we need this

Reply:
( marking discussion moved to Slack so it doesn't look like I'm ignoring the comment 😄 )